model/Core/Classes/Relationship.md

```diff
@@ -11,6 +11,12 @@ Describes a relationship between one or more elements.
 A Relationship is a grouping of characteristics unique to an assertion
 that one Element is related to one or more other Elements in some way.
 
+To explicitly assert that no such relationships exist, the `to` property 
+should contain the 'NONE' individual.
```

Comment:
```suggestion
should contain the 'NONE' individual and no other elements.
A relationship that contains 'NONE' and other elements in the `to` property is not valid.
```

Reply:
This looks like a good added clarification.